### PR TITLE
Restructure mgmt guides from Core docs to SDK docs - closes #731

### LINKED ISSUE #731: Restructure management guides from Core 2.0.0 to SDK 3.0.2
Some cross references to the SDK docs are missing and need to be added, including all associated links.

### PR DESCRIPTION
**Pull request by chris529**
Which content is outdated?
The content in the management guides is partly redundant with the guides in the SDK docs.

What should be the new content?
Update the Lisk Core documentation (e.g. add references to the Lisk SDK documentation) so that only Lisk Core specific topics are being covered in it.

For the general guides, refer to the Lisk SDK docs.

Closes #731

### CODE REVIEW

File: modules/ROOT/pages/index.adoc [hunk 1]
```diff
@@ -3,10 +3,14 @@ Mona Bärenfänger <mona@lightcurve.io>
 :description: The introduction page describes Lisk Core and the different networks, distributions and general technology stack of Lisk Core.
 :imagesdir: ./../assets/images
 :toc:
-:page-aliases: troubleshooting.adoc, getting-started.adoc
+:page-aliases: troubleshooting.adoc
 :page-no-previous: true
 :page-next: /lisk-core/interact-with-the-api.html
 :page-next-title: Interact with the API
+:page-no-previous: true
```
Comment (Tschakki) [inline modules/ROOT/pages/index.adoc]: Duplicate line, please remove

File: modules/ROOT/pages/index.adoc [hunk 2]
```diff
@@ -3,10 +3,14 @@ Mona Bärenfänger <mona@lightcurve.io>
 :description: The introduction page describes Lisk Core and the different networks, distributions and general technology stack of Lisk Core.
 :imagesdir: ./../assets/images
 :toc:
-:page-aliases: troubleshooting.adoc, getting-started.adoc
+:page-aliases: troubleshooting.adoc
```
Comment (Tschakki) [inline modules/ROOT/pages/index.adoc]: Why is `getting-started.adoc` being removed?

File: modules/ROOT/pages/index.adoc [hunk 3]
```diff
@@ -3,10 +3,14 @@ Mona Bärenfänger <mona@lightcurve.io>
...
 :page-no-previous: true
 :page-next: /lisk-core/interact-with-the-api.html
 :page-next-title: Interact with the API
+:page-no-previous: true
+:v_sdk: master
+:page-aliases: monitoring.adoc
+:page-aliases: migration.adoc
```
Comment (Tschakki) [inline modules/ROOT/pages/index.adoc]: Please remove these lines, no change in page aliases needed here.

File: modules/ROOT/pages/index.adoc [hunk 4]
```diff
@@ -163,7 +170,7 @@ A snapshot is a backup of the complete blockchain.
 It can be used to speed up the synchronization process, instead of having to validate all transactions starting from the genesis block to the current block height.
 Lisk provides official snapshots of the blockchain which can be found in the following link: {url_lisk_snapshots}
 
-How to both rebuild from a snapshot, and to create your own snapshots is explained in the respective section for each <<distributions, distribution>> of the Lisk Core.
+How to rebuild from a snapshot and to create your own snapshots is explained in the administration section for each xref:{}[distribution] of the Lisk Core.
```
Comment (Tschakki) [inline modules/ROOT/pages/index.adoc]: A link attribute is missing here

File: modules/ROOT/pages/index.adoc [hunk 5]
```diff
@@ -97,7 +104,7 @@ There are several advantages why one would wish to set up a node, and these are
 This is especially important when running an exchange and implementing LSK tokens.
 - To acquire full control in order to xref:{url_config}[configure] the node to your specific requirements.
 - To create your own xref:{url_snapshots}[snapshots][[Snapshots]] of the blockchain.
-- To have the option to xref:{url_config_forging}[forge] new blocks, (assuming you are an active xref:{url_config_forging}[delegate]).
+- To have the option to xref:{url_config_forging}[forge] new blocks, (assuming you are an active delegate)
```
Comment (Tschakki) [inline modules/ROOT/pages/index.adoc]:
```suggestion
- To have the option to xref:{url_config_forging}[forge] new blocks, (assuming you are an active delegate).
```

File: modules/ROOT/pages/interact-with-the-api.adoc [hunk 6]
```diff
@@ -5,21 +5,21 @@ Mona Bärenfänger <mona@lightcurve.io>
...
 :v_sdk: master
-:url_api: https://swagger.io/
 :url_curl: https://curl.haxx.se/
 :url_postman: https://www.getpostman.com/
 :url_lisk_wallet: https://lisk.io/wallet
 :url_github_explorer: https://github.com/LiskHQ/lisk-explorer
 :url_explorer: https://explorer.lisk.io
 :url_explorer_testnet: https://testnet-explorer.lisk.io
 
+:url_api: reference/api.adoc
```
Comment (Tschakki) [inline modules/ROOT/pages/interact-with-the-api.adoc]: The old link for `url_api` was correct, please restore.

File: modules/ROOT/pages/interact-with-the-api.adoc [hunk 7]
```diff
@@ -5,21 +5,21 @@ Mona Bärenfänger <mona@lightcurve.io>
...
 :url_maintain_node: index.adoc#node
 :url_elements_pkg_api: {v_sdk}@lisk-sdk::references/lisk-elements/api-client.adoc
 :url_elements: {v_sdk}@lisk-sdk::references/lisk-elements/index.adoc
 :url_elements_pkg: {v_sdk}@lisk-sdk::references/lisk-elements/index.adoc
 :url_commander: {v_sdk}@lisk-sdk::references/lisk-commander/index.adoc
 :url_commander_commands: {v_sdk}@lisk-sdk::references/lisk-commander/commands.adoc
-:url_config_api_access: management/configuration.adoc#api_access
+:url_config_api_access: {v_sdk}@lisk-sdk::guide/node-management/api-access.adoc
```
Comment (Tschakki) [inline modules/ROOT/pages/interact-with-the-api.adoc]:
```suggestion
:url_config_api_access: {v_sdk}@lisk-sdk::guides/node-management/api-access.adoc
```

File: modules/ROOT/pages/migration.adoc [hunk 8]
```diff
@@ -4,11 +4,11 @@ Mona Bärenfänger <mona@lightcurve.io>
 :toc:
 :page-previous: /lisk-core/management/configuration.html
 :page-previous-title: Configuration
+:v_sdk: master
```
Comment (Tschakki) [inline modules/ROOT/pages/migration.adoc]: No need to add this

File: modules/ROOT/pages/update/source.adoc [hunk 9]
```diff
@@ -7,6 +7,7 @@ Mona Bärenfänger <mona@lightcurve.io>
 :page-previous: /lisk-core/management/source.html
 :page-next-title: Monitoring
 :page-previous-title: Source code commands
+:v_sdk: master
```
Comment (Tschakki) [inline modules/ROOT/pages/update/source.adoc]: No need to add this here.

**Review by Tschakki — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/index.adoc (hunk 2 above)
Comment (chris529) [reply modules/ROOT/pages/index.adoc]: Ah yes...this was an error, have now fixed it 👍

File: modules/ROOT/pages/index.adoc (hunk 3 above)
Comment (chris529) [reply modules/ROOT/pages/index.adoc]: Done 👍

File: modules/ROOT/pages/index.adoc (hunk 4 above)
Comment (chris529) [reply modules/ROOT/pages/index.adoc]: Yes, indeed..It should be fixed now.

File: modules/ROOT/pages/interact-with-the-api.adoc (hunk 6 above)
Comment (chris529) [reply modules/ROOT/pages/interact-with-the-api.adoc]: OK it should be fine now 👍

File: modules/ROOT/pages/migration.adoc (hunk 8 above)
Comment (chris529) [reply modules/ROOT/pages/migration.adoc]: Yes, indeed this was an error as there is no reference to the SDK here. It is now removed 👍

File: modules/ROOT/pages/update/source.adoc (hunk 9 above)
Comment (chris529) [reply modules/ROOT/pages/update/source.adoc]: Same error as above..now corrected 👍

**Review by Tschakki — APPROVED** [review]
LGTM

**Pull request merged**